examples/resources/biganimal_cluster/single_node/bah_aws/resource.tf

```diff
@@ -0,0 +1,85 @@
+terraform {
```

Comment:
No principal ids?

Reply:
I added those values as commented out and created https://github.com/EnterpriseDB/terraform-provider-biganimal/issues/339.